agent/lib/kontena/workers/container_log_worker.rb

```diff
@@ -63,14 +63,14 @@ def start(since = 0)
     def on_message(id, stream, chunk)
       match = chunk.match(CHUNK_REGEX)
       return unless match
-      time = DateTime.parse(match[1])
+      time = match[1]
       data = match[2]
       msg = {
         id: id,
         service: @container.service_name,
         stack: @container.stack_name,
         instance: @container.instance_number,
-        time: time.utc.xmlschema,
```

Comment:
:+1: for standardizing on the use of ISO8601 timestamps for the agent-server RPC in general. This is currently a bit of a mess, with different parts of the RPC using different formats. Unfortunately some of those are also in the container labels, and harder to change.

There's also a subtle change here: `t.xmlschema` doesn't include any fractional seconds by default, whereas the Docker API log timestamps are like `2017-09-18T10:31:50.949628016Z`. The logs that show up on the CLI now include sub-second timestamps as a result of this change:

```
2017-09-18T08:11:02.000Z [1]: Sep 18 08:11:02 localhost haproxy[52]: 10.81.0.1:41864 [18/Sep/2017:08:11:01.981] http-in http-in/<NOSRV> -1/-1/-1/-1/28 400 187 - - CR-- 0/0/0/0/3 0/0 "<BADREQ>"
2017-09-18T08:11:02.000Z [1]: Sep 18 08:11:02 localhost haproxy[52]: 10.81.0.1:41864 [18/Sep/2017:08:11:01.981] http-in http-in/<NOSRV> -1/-1/-1/-1/28 400 187 - - CR-- 0/0/0/0/3 0/0 "<BADREQ>"
2017-09-18T10:31:50.949Z [1]: Sep 18 10:31:50 localhost haproxy[52]: 10.81.0.1:43290 [18/Sep/2017:10:31:50.769] http-in http-in/<NOSRV> -1/-1/-1/-1/7 400 187 - - CR-- 0/0/0/0/3 0/0 "<BADREQ>"
2017-09-18T10:41:47.400Z [1]: Sep 18 10:41:47 localhost haproxy[52]: 10.81.0.1:43434 [18/Sep/2017:10:41:47.395] http-in http-in/<NOSRV> -1/-1/-1/-1/3 400 187 - - CR-- 0/0/0/0/3 0/0 "<BADREQ>"
```

Reply:
BTW: using `t.xmlschema` needs an optional parameter to include fractional seconds... this should ideally be some `MessagePack`-level thing :/

```
irb(main):004:0> t.utc.xmlschema
=> "2017-09-18T10:37:12Z"
irb(main):006:0> t.utc.xmlschema(6)
=> "2017-09-18T10:37:12.226407Z"
```